tests/vtr/lutff-pair/omux/omux.pb_type.xml

```diff
@@ -0,0 +1,38 @@
+<pb_type name="omux" num_pb="1">
+  <metadata>
+    <meta name="type">bel</meta>
+    <meta name="subtype">routing</meta>
+  </metadata>
+  <!--
```

Comment:
What is going on here?

Reply:
This XML was generated using `mux_gen.py`. It inserts a comment to XML but apparently when invoked as a Python function it was doing that incorrectly. I fixed that.